src/plugins/s390.c

```diff
@@ -598,25 +603,26 @@ gboolean bd_s390_zfcp_online (const gchar *devno, const gchar *wwpn, const gchar
     /* part 01: make sure device is available/not on device ignore list */
     fd = fopen (online, "r");
     if (!fd) {
-        boolrc = bd_utils_exec_and_report_error_no_progress (zfcp_cio_free, NULL, error);
-
+        boolrc = bd_utils_exec_and_report_error_no_progress (zfcp_cio_free, NULL, NULL);
```

Comment:
We were leaking errors on multiple places in the s390 plugin. I was thinking about fixing this by prefixing the leaking error instead of overwriting it by `g_set_error`, but in the end I decided to just keep ignoring the error from the calls and not changing the logic here.

Reply:
I think this looks correct. The detailed error message might be lost, not a big deal I guess.